src/components/users/validation-summary/utils.ts

```diff
@@ -0,0 +1,66 @@
+import { EventManager } from "@paperbits/common/events";
+import { ValidationReport } from "../../../contracts/validationReport";
+
+export const onValidationErrors = "onValidationErrors";
+
+export enum errorSources {
```

Comment:
I guess we could move it to separate file. Also, please use capital case for enums.